deploy/docker/jupyterhub/Dockerfile

```diff
@@ -1,13 +1,39 @@
-FROM jupyterhub/jupyterhub:1.0.0
+FROM ubuntu:18.04
+LABEL maintainer="Labshare <konstantin.taletskiy@labshare.org>"
 
 # Track semantic versioning
 COPY VERSION /
+COPY stacks stacks
```

Comment:
Can you add a comment to indicate what is expected to be copied into the Docker image?

Reply:
Done